docs/ref/models/fields.txt

```diff
@@ -1009,7 +1009,7 @@ periodically via e.g. cron).
 .. class:: FilePathField(path=None, match=None, recursive=False, max_length=100, **options)
 
 A :class:`CharField` whose choices are limited to the filenames in a certain
-directory on the filesystem. Has three special arguments, of which the first is
+directory on the filesystem. Has some special arguments, of which the first is
```

Comment:
This change is not related with the main change. I moved it to a separate commit and added signature fix.

Reply:
Thanks